### DISCUSSION TITLE
Alternative Title Selection

### DISCUSSION THREAD

**Discussion by watermelvin** [KRTirtho/spotube#352] [Q&A] [answered]
I've seen from some of the other issues that theres some kind of alternative title selection feature when the default song match is incorrect, but I can't seem to find the option anywhere on either the latest release from flathub or the latest release from fdroid (both 2.6.0). If this feature does in fact exist, maybe making it more obvious would be a good idea, as I haven't seen it in the settings, and there doesn't seem to be an option for it in the search ui either.

**Comment by KRTirtho** [comment, marked as answer]
This is the icon. I think the Icon itself is already pretty descriptive, thus I didn't bother to make it more specific. But looks like that is not the case. I think an start-up _Introduction/Guide_ UI for Spotube is really needed

![image](https://user-images.githubusercontent.com/61944859/208582957-3f31453c-ee5b-4907-9186-c7bedcd4c813.png)
